src/cdk/a11y/key-manager/list-key-manager.ts

```diff
@@ -280,6 +282,22 @@ export class ListKeyManager<T extends ListKeyManagerOption> {
           return;
         }
 
+      case PAGE_UP:
+        if (isModifierAllowed) {
+          this.setFirstItemActive();
```

Comment:
Page up/down aren't supposed to focus the first/last items, but rather move one page up/down in the list. The reason why we haven't implemented it in the `ListKeyManager` is that it doesn't know how large the list is and the offset at which focus is set currently.

Reply:
Hi, if you take a look at my W3C link, it is possible to go 10 elements down/up or select the first/last option for the implentation of page_up/page_down. Or did I udnerstand somethin wrong from this reference implementation?

Reply:
The number 10 seems a little arbitrary to me, my understanding is that at least in native applications, it's supposed to go up/down by a page (hence the name).

Reply:
I definitly understand your concerns. I think the idea behing is, that a combobox is an "popup" which is after opening handled like it is a own page => therefore the jump of 10 (or last/first).

If you like I can implement the jump for 10 items, or we can forget the PR. I only saw it is an easy fix and thought i contribute it :)

Reply:
I think that's fine, but we should make it configurable, similar to how it's done for the home/end keys. E.g. something with `ListKeyManager.withPageUpDown(10)`.

Reply:
I pushed some tests and the integration with jumping 10 or to first/last one! Are there sth missing?